package.json

```diff
@@ -25,9 +25,10 @@
   "homepage": "https://github.com/odysseyscience/react-s3-uploader",
   "dependencies": {
     "aws-sdk": "2.x",
+    "koa-router": "^5.4.0",
```

Comment:
Im not familiar with koa, but notice we took out the `express` dependency in this file, and are just relying on people have it installed if they are using the bundled router.  That keeps it so that people not using `express` (or in this case `koa`) don't have to download the dependency.

Is koa a large library?  Does it have a lot of dependencies?  If so, we should probably remove it from this file.